summoner-cli/examples/cabal-full/.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           - "8.4.4"
           - "8.6.5"
           - "8.8.4"
-          - "8.10.2"
+          - "8.10.3"
```

Comment:
It is so nice to see all the updates and refreshments to Summoner 😊 
Thanks for this!